lib/screens/login/login_screen.dart

```diff
@@ -10,8 +11,86 @@ class LoginScreen extends StatefulWidget {
 }
 
 class _LoginScreenState extends State<LoginScreen> {
+  String error = '', password, email;
+  bool showPassword = false;
+  bool loading = false;
+  final GlobalKey<FormState> formKey = GlobalKey<FormState>();
+
+  onChangePassword(String value) {
+    setState(() {
+      password = value.trim();
+    });
+  }
+
+  onChangeEmail(String value) {
+    setState(() {
+      email = value.trim();
+    });
+  }
+
+  onToggleShowPassword() {
+    setState(() {
+      showPassword = !showPassword;
+    });
+  }
+
+  onClickSignInButton(context) async {
```

Comment:
context is already a property in StatefulWidget

```suggestion
  onClickSignInButton() async {
```

Reply:
In build method, 
```
onClickSignInButton: () {
              onClickSignInButton(context);
            },
```
can be written as
```
onClickSignInButton: onClickSignInButton,
```